### PR TITLE
[W5.7] [T13-3] Hansel

### PR DESCRIPTION
**Pull request by handshou**
Add Sort command.
TODO: JUnit tests for Sort.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -88,4 +88,8 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person p) {
+        return getName().toString().compareTo(p.getName().toString());
```
Comment (NancyQuris) [inline src/seedu/addressbook/data/person/Person.java:93]: Good job Hansel! I can see the effort of adding a new sort method that sort according to name.

**Comment by Gongjie663746** [comment]
Good job Hansel! Thank you for the effort!

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+
+public class SortCommand extends Command {
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/SortCommand.java:5]: Add descriptive comments for public classes

File: src/seedu/addressbook/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static final String COMMAND_WORD = "sort";
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/SortCommand.java:7]: Avoid public class variables

File: src/seedu/addressbook/commands/SortCommand.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the address book. "
+            + "Sorting takes place using alphabetical order.\n";
+
+    public static final String MESSAGE_SUCCESS = "Address book sorted.";
+
+    public SortCommand() throws Exception {
+
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/SortCommand.java:15]: Incomplete implementation?

File: src/seedu/addressbook/commands/SortCommand.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the address book. "
+            + "Sorting takes place using alphabetical order.\n";
+
+    public static final String MESSAGE_SUCCESS = "Address book sorted.";
+
+    public SortCommand() throws Exception {
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/SortCommand.java:14]: Public method description?

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 6]
```diff
@@ -10,6 +10,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void elementsContainNull() throws Exception {
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:14]: This commit shouldn't be part of this PR. In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the master branch that is in sync with the upstream master branch

**Review by parvathy-sudhir-pillai — APPROVED** [review]
Good effort on this PR @handshou. 

Suggestions to improve further:
* update user and dev docs
* update I/O and jUnit tests to test the new functionality
* limit the PR to one enhancement and avoid including changes not related to that enhancement or unmerged code from other PRs
* Pay closer attention to the coding standard

Please close the PR after viewing comments.

**Comment by handshou** [comment]
Thanks Ms Parvathy!
